### PR TITLE
[ACTION] expofp #4596

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

**Review by GTFalcao — CHANGES_REQUESTED** [review]
Hi @lcaresia , thanks for your contribution!

I've left some very minor suggestions and a few component versions that need to be reset.

File: components/expofp/actions/update-booth/update-booth.mjs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    const response = await this.expofp.updateBooth({
+      $,
+      data: {
+        eventId: this.eventId,
+        name: this.boothId,
+        description: this.description,
+      },
+    });
+
+    $.export("$summary", `Successfully updasted booth with ID ${this.boothId}`);
```
Comment (GTFalcao) [inline components/expofp/actions/update-booth/update-booth.mjs:39]: Typo fix
```suggestion
    $.export("$summary", `Successfully updated booth with ID ${this.boothId}`);
```

**Review by GTFalcao — COMMENTED** [review]
LGTM! 👏 

There's just a minor typo in a summary, so I'm moving this to Ready for QA.

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[ExpoFx.pdf](https://github.com/PipedreamHQ/pipedream/files/9909020/ExpoFx.pdf)

**Review by GTFalcao — APPROVED** [review]

**Pull request merged**